app/service/init.go

```diff
@@ -0,0 +1,51 @@
+package service
+
+import (
+	"os"
+	"tat_gogogo/infrastructure/router"
+	"tat_gogogo/utilities/logs"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-contrib/pprof"
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+)
+
+/*
+configInit is init load config.
+*/
+func configInit() {
+	viper.SetConfigName("config")
+	viper.AddConfigPath("..")
+	err := viper.ReadInConfig()
+	if err != nil {
+		panic("Fatal error config file: " + err.Error())
+	}
+}
+
+/*
+ginInit is create gin engine and register middleware.
+*/
+func ginInit() *gin.Engine {
+	ginRouter := gin.Default()
+	// -----------------------------------------------[Middleware]
```

Comment:
@dennySORA, we actually don't do this I'd suggest you remove those comment and write your thought on the vary top of comment.

Reply:
OK,i like use this.
but this is your project,so i change it.